src/app/component/shared/shared.module.ts

```diff
@@ -13,6 +13,7 @@ import {
 } from './components';
 import { MatDialogModule } from '@angular/material/dialog';
 import { MatCheckboxModule } from '@angular/material/checkbox';
+import {MatProgressSpinnerModule} from '@angular/material/progress-spinner';
```

Comment:
Please, add spaces like in other imports

Reply:
done